storage/rocksdb/ha_rocksdb.cc

```diff
@@ -7432,6 +7443,8 @@ static int rocksdb_init_internal(void *const p) {
   rocksdb_hton->clone_interface.clone_apply = rocksdb_clone_apply;
   rocksdb_hton->clone_interface.clone_apply_end = rocksdb_clone_apply_end;
 
+  rocksdb_hton->dict_register_dd_table_id = rocksdb_dict_register_dd_table_id;
```

Comment:
Let's do `handler::get_se_private_data` skeleton implementation too that ignores its 1st arg but clears the set if the 2nd arg is true, but add an `assert(false);` as well since the implementation is not complete